src/pages/cart.js

```diff
@@ -58,10 +60,17 @@ const styles = (theme) => ({
 });
 
 @trackProductViewed()
+@inject("uiStore")
+@observer
 @withStyles(styles)
 class CartPage extends Component {
   static propTypes = {
-    classes: PropTypes.object
+    classes: PropTypes.object,
+    shop: PropTypes.shape({
+      currency: PropTypes.shape({
```

Comment:
Why do we include this `currency` shape here? We don't seem to use it anywhere.

We do use `name` and `description` under `shop` though, should those fields be here instead?